### PR TITLE
Expose `progress_bar` parameter for `prefetch_series_values`

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: src/neptune_fetcher/cache.py [hunk 1]
```diff
@@ -53,10 +56,12 @@ def cache_miss(self, paths: List[str]) -> None:
     def prefetch(self, paths: List[str]) -> None:
         self.cache_miss(paths)
 
-    def prefetch_series_values(self, paths: List[str], use_threads: bool) -> None:
+    def prefetch_series_values(
+        self, paths: List[str], use_threads: bool, progress_bar: "ProgressBarType" = None
```
Comment (SiddhantSadangi) [inline src/neptune_fetcher/cache.py:60]: Why `"ProgressBarType"` and not just `ProgressBarCallback`?

File: src/neptune_fetcher/read_only_run.py [hunk 2]
```diff
@@ -114,13 +116,17 @@ def prefetch(self, paths: List[str]) -> None:
         """
         self._cache.prefetch(paths=paths)
 
-    def prefetch_series_values(self, paths: List[str], use_threads: bool = False) -> None:
+    def prefetch_series_values(
+        self, paths: List[str], use_threads: bool = False, progress_bar: "ProgressBarType" = None
```
Comment (SiddhantSadangi) [inline src/neptune_fetcher/read_only_run.py:120]: Same as before

**Review by SiddhantSadangi — COMMENTED** [review]
The test changes look fine by themselves, but I couldn't find where `prepare_runs` is being used

**Comment by Raalsky** [comment]
> The test changes look fine by themselves, but I couldn't find where `prepare_runs` is being used

Manually 😢

**Review by SiddhantSadangi — APPROVED** [review]

**Review by AleksanderWWW — APPROVED** [review]

**Pull request merged**